src/main/java/edu/kis/vh/nursery/stack/IntArrayStack.java

```diff
@@ -0,0 +1,37 @@
+package edu.kis.vh.nursery.stack;
+
+public class IntArrayStack implements StackInterface {
+		private static final int FULL_STACK_VALUE = 11;
+		private static final int STACK_CAPACITY = 12;
+		private int[] numbers = new int[STACK_CAPACITY];
+		private int total = EMPTY_STACK_VALUE;

```

Comment:
ERROR: total variable initialization and validation should always remain -1 (regardless of returned EMPTY_STACK_VALUE).